[2022-05-23 20:15:30,860] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-16T00:00:00+00:00 [queued]>
[2022-05-23 20:15:30,869] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-16T00:00:00+00:00 [queued]>
[2022-05-23 20:15:30,870] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:15:30,870] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-05-23 20:15:30,870] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:15:30,884] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): second_task> on 2021-12-16 00:00:00+00:00
[2022-05-23 20:15:30,891] {standard_task_runner.py:52} INFO - Started process 2000 to run task
[2022-05-23 20:15:30,894] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-16T00:00:00+00:00', '--job-id', '421', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpvojd0hnq', '--error-file', '/tmp/tmps34r5ox4']
[2022-05-23 20:15:30,895] {standard_task_runner.py:80} INFO - Job 421: Subtask second_task
[2022-05-23 20:15:30,961] {task_command.py:369} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-16T00:00:00+00:00 [running]> on host 061a475e6c14
[2022-05-23 20:15:31,077] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=gerson
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-16T00:00:00+00:00
[2022-05-23 20:15:31,078] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-23 20:15:31,078] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-05-23 20:15:31,094] {subprocess.py:85} INFO - Output:
[2022-05-23 20:15:31,096] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-05-23 20:15:31,096] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-23 20:15:31,143] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211216T000000, start_date=20220523T201530, end_date=20220523T201531
[2022-05-23 20:15:31,190] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-23 20:15:31,256] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
